cmd/up/up.go

```diff
@@ -364,13 +365,21 @@ func Up() *cobra.Command {
 }
 
 // AddArgs sets the args as options and return err if it's not compatible
-func (o *UpOptions) AddArgs(args []string) error {
+func (o *UpOptions) AddArgs(args []string, oktetoManifest *model.Manifest) error {
 
 	if len(args) == 1 {
-		o.DevName = args[0]
+		if _, ok := oktetoManifest.Dev[args[0]]; ok {
+			o.DevName = args[0]
+		} else {
+			o.commandToExecute = args
+		}
 	} else if len(args) > 1 {
-		o.DevName = args[0]
-		o.commandToExecute = args[1:]
+		if _, ok := oktetoManifest.Dev[args[0]]; ok {
+			o.DevName = args[0]
+			o.commandToExecute = args[1:]
+		} else {
+			o.commandToExecute = args
+		}
 	}
 
```

Comment:
Can we add a log here to know what's the content of both values? This will help us to debug whenever there is a problem
```suggestion
oktetoLog.Infof("dev name: %s / commandToExecute: %v", o.DevName, o.commandToExecute)
```